um_canonicalize/tags.yaml

```diff
@@ -244,6 +278,9 @@ categories:
     - '3'
     - '4'
     - INCL
+    - 1+INCL
```

Comment:
Probably makes sense to write `1INCL` instead since, assuming this is first person inclusive, it's not merely first person + inclusive but rather "inclusive(1st person)".

Reply:
Yeah, but this is in the spec.